terraform_files/variables-libvirt.tf

```diff
@@ -20,12 +20,32 @@ variable "cluster_domain" {
 
 variable "machine_cidr" {
```

Comment:
don't see usage if it anymore

Reply:
Some of them are populated by scripts and then applied. Terraform had been complaining until we defined them in variables-libvirt.tf.

Reply:
https://github.com/openshift/assisted-test-infra/blob/b54bd3829f6c4e83e6f1cacee481b81b0fe338d7/discovery-infra%2Ftest_infra%2Fcontrollers%2Fnode_controllers%2Fterraform_controller.py#L41

Reply:
but i don't see any usage of machine_cidr in terraform, so why to leave

Reply:
but you deleted it from main.tf so u need to change all the places that are using it and change them and to delete the old variables. Right?

Reply:
Done